osi_logicallane.proto

```diff
@@ -822,5 +828,123 @@ message LogicalLane
         //
         optional double end_s_other = 5;
     }
+
+    //
+    // Describes traffic rules on a lane. 
+    // The traffic rule can thereby be induced by regulations, traffic signs 
+    // or by other means. If the modeled traffic rule is induced by a traffic sign
+    // the information should be identical with the respective traffic sign.
+    //
+    // Note: Every instance should be corresponding to only one specific rule.
+    // The type of the traffic rule should be set using the respective field.
+    // Additionally, every message should contain the traffic rule validity information
+    // and the respective field for the respective traffic rule type.
+    // In case of traffic rule (priority) conflicts for rules of the same type that 
+    // can not be depicted using the traffic rule validity only the currently 
+    // valid rule should be provided.
+    //
+    // Note: Each traffic rule corresponds to only one lane. If the traffic rule
```

Comment:
Maybe a theoretical concern, but at least worth mentioning: Currently I see no way of defining for which travel direction the rule is valid? Speed limits are generally directional (valid for only one travel direction), if they are defined by signs. What happens if there is a `bidirectional` lane with different rules for different traffic directions?

OK, so hopefully noone is crazy enough to do something like this, and maybe OSI does not need to support this. But I thought I should at least mention it :)

Reply:
To be handled by requiring the direction of travel to match the start_s/end_s order of the TrafficRuleValidity, see below.